.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,19 @@
-# Pull Request
-
-## [Link to story in trello](#)
+# [Story in Trello](#)
 
 ## Done?
 
-- [ ] Acceptance criterias are met
-- [ ] Code quality checks are green
-- [ ] Readme updated if needed
-- [ ] Story under test
-- [ ] Mobile works
-- [ ] Seeds created
-- [ ] Translated
-- [ ] Code is reviewed
+### Done for approval?
+
+* [ ] Acceptance criteria are met
+* [ ] Code quality checks are green
+* [ ] Readme updated if needed
+* [ ] Story under test
+* [ ] Mobile works
+* [ ] Seeds created
+* [ ] Translated
+* [ ] Code is reviewed
```

Comment:
Code review is already enforced by GitHub, maybe we can remove this point? Also if you like I can add Rubocop to the Travis setup, so we can remove the code quality checkbox as well.